source/includes/gcp/_k8_charts.md

```diff
@@ -0,0 +1,315 @@
+        },
+        "repoUrl": "https://hub.helm.sh/api/chartsvc",
+        "description": "A Helm chart for Aerospike in Kubernetes",
+        "home": "http://aerospike.com",
+        "keywords": [
+          "aerospike",
+          "big-data"
+        ],
+        "maintainers": [
+          {
+            "name": "kavehmz",
+            "email": "kavehmz@gmail.com"
+          },
+          {
+            "name": "okgolove",
+            "email": "okgolove@markeloff.net"
+          }
+        ],
+        "sources": [
+          "https://github.com/aerospike/aerospike-server"
+        ],
+        "icon": "/v1/assets/stable/aerospike/logo"
+      },
+      "links": {
+        "self": "/v1/charts/stable/aerospike"
+      },
+      "relationships": {
+        "latestChartVersion": {
+          "data": {
+            "version": "0.3.2",
+            "digest": "97cfd56a20129d65b30de9d77267bf30f7b5f2d28565285282e16b26678a4ce3",
+            "urls": [
+              "https://kubernetes-charts.storage.googleapis.com/aerospike-0.3.2.tgz"
+            ],
+            "readme": "/v1/assets/stable/aerospike/versions/0.3.2/README.md",
+            "values": "/v1/assets/stable/aerospike/versions/0.3.2/values.yaml"
+          },
+          "links": {
+            "self": "/v1/charts/stable/aerospike/versions/0.3.2"
+          }
+        }
+      },
+      "insecureServer": false
+    },
+    {
+      "id": "stable/airflow",
+      "type": "chart",
+      "attributes": {
+        "name": "airflow",
+        "repo": {
+          "name": "stable",
+          "url": "https://kubernetes-charts.storage.googleapis.com"
+        },
+        "repoUrl": "https://hub.helm.sh/api/chartsvc",
+        "description": "Airflow is a platform to programmatically author, schedule and monitor workflows",
+        "home": "https://airflow.apache.org/",
+        "keywords": [
+          "workflow",
+          "dag"
+        ],
+        "maintainers": [
+          {
+            "name": "gsemet",
+            "email": "gaetan@xeberon.net"
+          }
+        ],
+        "sources": [
+          "https://airflow.apache.org/"
+        ],
+        "icon": ""
+      },
+      "links": {
+        "self": "/v1/charts/stable/airflow"
+      },
+      "relationships": {
+        "latestChartVersion": {
+          "data": {
+            "version": "5.2.5",
+            "digest": "de2c99e9f7b277be11915229381bb11ab68f875379eebdfd450c64d307c9bb29",
+            "urls": [
+              "https://kubernetes-charts.storage.googleapis.com/airflow-5.2.5.tgz"
+            ],
+            "readme": "/v1/assets/stable/airflow/versions/5.2.5/README.md",
+            "values": "/v1/assets/stable/airflow/versions/5.2.5/values.yaml"
+          },
+          "links": {
+            "self": "/v1/charts/stable/airflow/versions/5.2.5"
+          }
+        }
+      },
+      "insecureServer": false
+    }
+  ],
+  "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
```

Comment:
that's a big example XD

Reply:
These are simplified real example. We can talk on how we can make them shorter.

Reply:
fixed